src/firefly/js/core/LayoutCntlr.js

```diff
@@ -58,12 +59,24 @@ export const UPDATE_LAYOUT     = `${LAYOUT_PATH}.updateLayout`;
 export const UPDATE_GRID_VIEW  = `${LAYOUT_PATH}.updateGridView`;
 export const SET_LAYOUT         = `${LAYOUT_PATH}.setLayout`;
 export const SET_LAYOUT_MODE    = `${LAYOUT_PATH}.setLayoutMode`;
+export const TRIVIEW_LAYOUT    = `${LAYOUT_PATH}.triviewLayout`;
 export const SHOW_DROPDOWN      = `${LAYOUT_PATH}.showDropDown`;
 export const ADD_CELL           = `${LAYOUT_PATH}.addCell`;
 export const REMOVE_CELL        = `${LAYOUT_PATH}.removeCell`;
 export const ENABLE_SPECIAL_VIEWER= `${LAYOUT_PATH}.enableSpecialViewer`;
 export const MENU_UPDATE      = `${LAYOUT_PATH}.menuUpdate`;
 
+
+export const TRIVIEW_IC_C_T= 'TRIVIEW_IC_C_T'; //top left: image/cov, top right: charts, bottom: tables
+export const TRIVIEW_I_CC_T= 'TRIVIEW_I_CC_T';//top left: image, top right: charts/cov, bottom: tables
+export const BIVIEW_IC_C= 'BIVIEW_IC_C'; //left: image/cov, right: charts
```

Comment:
[not a big deal] I found this bit confusing without looking at desc - might consider renaming Charts' `C` to `Ch`

Reply:
good idea